src/gui/graphdesignerpage.h

```diff
@@ -25,19 +25,20 @@
 #include <QStringList>
 
 #include "core/include/enum.h"
+#include "core/mainapp.h"
+#include "core/graphinputs.h"
 
 #include "maingui.h"
 #include "abstractgraph.h"
 #include "attributerange.h"
-
-#include "core/mainapp.h"
-#include "core/graphinputs.h"
+#include "fullinspector.h"
```

Comment:
you should add a forward declaration OR include the header
in thise case, you could move this `#include "fullinspector.h"` to the cpp file